osclib/stagingapi.py

```diff
@@ -1004,10 +1004,6 @@ def project_status(self, staging=None, aggregate=False):
 
     def project_status_walk(self, project):
         yield project
-        for sub in project['subprojects']:
```

Comment:
if there are no subprojects, you don't need a walk

Reply:
I also see some other places need to update, will re-do this PR